### PR TITLE
[presets] drop --verify from RHOCP preset, call "ostree fsck" under plugin specific option

### PR DESCRIPTION
**Pull request by pmoravec**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

File: sos/report/plugins/ostree.py [hunk 1]
```diff
@@ -17,6 +17,9 @@ class OSTree(Plugin, IndependentPlugin):
     profiles = ('system', 'sysmgmt', 'packagemanager')
     files = ('/ostree',)
     services = ('ostree-finalize-staged', 'ostree-boot-complete')
+    option_list = [
+        PluginOpt('fsck', default=False, desc='collect ostree fsck')
```
Comment (TurboTurtle) [inline sos/report/plugins/ostree.py:21]: Unittests are failing since `PluginOpt` is not imported in this plugin.

**Review by TurboTurtle — COMMENTED** [review]
Ack to the option change. Note on the test suite below.

File: sos/report/plugins/ostree.py (hunk 1 above)
Comment (pmoravec) [reply sos/report/plugins/ostree.py:21]: I was blind, thanks.

**Review by TurboTurtle — APPROVED** [review]
Ack

**Pull request merged**